packages/components/src/toggle-group-control/toggle-group-control/component.tsx

```diff
@@ -44,10 +46,25 @@ function UnconnectedToggleGroupControl(
 		...otherProps
 	} = useContextSystem( props, 'ToggleGroupControl' );
 
-	const baseId = useInstanceId( ToggleGroupControl, 'toggle-group-control' );
 	const normalizedSize =
 		__next40pxDefaultSize && size === 'default' ? '__unstable-large' : size;
 
+	const ref = useRef< HTMLDivElement >();
+	const [ activeElement, setActiveElement ] = useState< HTMLElement >();
+	useLayoutEffect( () => {
+		setActiveElement(
+			ref.current?.querySelector( `button[value="${ value }"]` )
+				?.parentElement ?? undefined
+		);
+	}, [ value ] );
```

Comment:
Nit: Wondering if relying on `querySelector` and `parentElement`is too brittle, as it could break if the underlying DOM tree changes.

An alternative could be to expose `setActiveElement` via internal context to the option sub-components, and have them call `setActiveElement` with the relevant HTML element.

Reply:
Done in fe2b51e9f17be939d789c4f7dc14ec6771876702, it fixes the `isDeselectable` as well ✨